bittensor/utils/weight_utils.py

```diff
@@ -341,3 +346,46 @@ def process_weights_for_netuid(
     bittensor.logging.debug("final_weights", normalized_weights)
 
     return non_zero_weight_uids, normalized_weights
+
+
+def generate_weight_hash(
+    who: str, netuid: int, uids: List[int], values: List[int], version_key: int
```

Comment:
Let's not name this `who`. I want to see more consistency in naming. Let's say `ss58_address`, if it's supposed to be the ss58 address.

Reply:
+1